### PR TITLE
make flutter doctor -v work

### PR DESCRIPTION
**Pull request by devoncarew**
- make `flutter doctor -v` work (previously, only `flutter -v doctor` would print the verbose doctor output)

@jcollins-g

### CODE REVIEW

File: packages/flutter_tools/lib/executable.dart [hunk 1]
```diff
@@ -38,8 +38,7 @@ import 'src/runner/flutter_command.dart';
 Future<Null> main(List<String> args) async {
   final bool verbose = args.contains('-v') || args.contains('--verbose');
 
-  final bool doctor = (args.isNotEmpty && args.first == 'doctor') ||
-      (args.length == 2 && verbose && args.last == 'doctor');
+  final bool doctor = args.contains('doctor');
```
Comment (jcollins-g) [inline packages/flutter_tools/lib/executable.dart]: I think this is too broad;  any parameter named "doctor" will trigger the flutter doctor here.

Comment (jcollins-g) [reply packages/flutter_tools/lib/executable.dart]: as an example:  "flutter create doctor" will have the side effect of muting the command logging, which is probably not what you want.

Comment (devoncarew) [reply packages/flutter_tools/lib/executable.dart]: Hmm, we could explicitly check for `doctor`, `-v doctor`, and `doctor -v`.

**Comment by devoncarew** [comment]
I'm going to close this for now; we may want to explore unifying the output of `flutter doctor` and `flutter doctor -v` - that would eliminate one place users can trip up (omitting or not knowing to run `-v`).

**Pull request closed without merging**